spec/ruby/core/class/inherited_spec.rb

```diff
@@ -98,4 +99,16 @@ class << top; protected :inherited; end
     -> { Class.new(top) }.should_not raise_error
   end
 
+  it "is invoked after the class is named when using class definition syntax" do
+    CoreClassSpecs::Callbacks::Child::INHERITED_NAME.should == "CoreClassSpecs::Callbacks::Child"
+  end
+
+  ruby_version_is "3.5" do
```

Comment:
```suggestion
  ruby_version_is "3.5" do # https://bugs.ruby-lang.org/issues/21143
```
Seems nice to find out more about this change :)